src/Contract/Operation/Flattenable.php

```diff
@@ -22,7 +22,7 @@ interface Flattenable
     /**
      * Flatten a collection of items into a simple flat collection.
      *
-     * @return Collection<TKey, T>
+     * @return Collection<mixed, mixed>
```

Comment:
unfortunately for `flatten` and `unwrap` these cannot be more specifically typed because the final collection will vary wildly depending on the shape of the data it had in it before applying these operations